src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/RegisterPage.java

```diff
@@ -42,7 +45,11 @@
 
   private final String mainWindowHandle;
 
-  public RegisterArea(boolean waitForNewWindow) {
+  public RegisterPage() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)